<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpclientWidget</class>
 <widget class="QWidget" name="GpclientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlobalProtect VPN</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="gatewayLabel">
     <property name="text">
      <string>Gateway:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="gatewayLineEdit">
     <property name="toolTip">
      <string>VPN server hostname or IP address (e.g., vpn.company.com)</string>
     </property>
     <property name="placeholderText">
      <string>vpn.company.com</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="browserLabel">
     <property name="text">
      <string>Browser:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="browserComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Browser for 2FA/SAML authentication. Select 'Custom...' to specify your own executable.</string>
     </property>
     <!-- Browser items are added programmatically in constructor to check file existence -->
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="dnsLabel">
     <property name="text">
      <string>DNS Servers:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dnsLineEdit">
     <property name="toolTip">
      <string>Recommended: Leave empty to use DNS servers from VPN with automatic split DNS configuration.
Split DNS allows resolving VPN-specific domains through VPN DNS while using local DNS for other domains.
Override: Enter semicolon-separated DNS servers (e.g., 8.8.8.8;8.8.4.4) to use custom DNS instead.</string>
     </property>
     <property name="placeholderText">
      <string>Leave empty for automatic VPN DNS (recommended)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hipLabel">
     <property name="text">
      <string>HIP:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="hipCheckBox">
     <property name="text">
      <string>Enable Host Integrity Protection (HIP)</string>
     </property>
     <property name="toolTip">
      <string>Enable HIP (Host Integrity Protection) to send host information to the VPN gateway. This is required by some organizations for security compliance.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
